lib/stm32/f3/flash.c

```diff
@@ -50,14 +50,105 @@
 /*---------------------------------------------------------------------------*/
 /** @brief Clear All Status Flags
 
-Clears program error, end of operation, busy flags.
-*/
+ Program error, end of operation, write protect error, busy.
+ */
```

Comment:
I undertand adding write protect, but why remove the fairly important "clears" verb?